src/components/Dashboard/index.js

```diff
@@ -73,283 +127,488 @@ const Dashboard = () => {
+                    fullWidth
+                    autoComplete="handle"
+                    required
+                    style={{ marginBottom: "15px" }}
+                    InputProps={{
+                      startAdornment: (
+                        <InputAdornment position="start">
+                          <PersonIcon style={{ color: "rgba(0,0,0,.25)" }} />
+                        </InputAdornment>
+                      ),
+                    }}
+                  />
+                  <TextField
+                    error={handleValidateError}
+                    label="User Handle"
+                    variant="outlined"
+                    placeholder="User Handle"
+                    value={handle}
+                    onChange={(event) => onChangeHandle(event.target.value)}
+                    helperText={
+                      handleValidateError ? handleValidateErrorMessage : null
+                    }
+                    fullWidth
+                    autoComplete="handle"
+                    required
+                    onFocus={onFocusHandle}
+                    style={{ marginBottom: "15px" }}
+                    InputProps={{
+                      startAdornment: (
+                        <InputAdornment position="start">
+                          <PersonOutlineIcon
+                            style={{ color: "rgba(0,0,0,.25)" }}
+                          />
+                        </InputAdornment>
+                      ),
+                    }}
+                  />
+                  <FormControl
+                    variant="outlined"
+                    error={countryValidateError}
+                    fullWidth
+                  >
+                    <InputLabel>User Country</InputLabel>
+                    <Select
+                      label="User Country"
+                      children={children}
+                      style={{ width: "100%" }}
+                      value={country}
+                      onChange={(event) => onChangeCountry(event.target.value)}
+                    ></Select>
+                  </FormControl>
+                </Box>
+                <Divider></Divider>
+                <Box m={3}>
+                  <Button
+                    size="small"
+                    fullWidth
+                    variant="contained"
+                    color="primary"
+                    style={{
+                      backgroundColor: "#3cafa9",
+                    }}
+                    onClick={() => setShowOrgForm(!showOrgForm)}
                   >
-                    <Input
-                      prefix={
-                        <UserAddOutlined style={{ color: "rgba(0,0,0,.25)" }} />
+                    {showOrgForm === false
+                      ? "I want to create an organization"
+                      : showOrgForm === true
+                      ? "I don't want to create an organization"
+                      : "I want to create an organization"}
```

Comment:
@Niweera, need your expert opinion on this piece of code in here. Not sure what kind of logic that is meant to be implemented.

Reply:
If I can remember correctly, it allows the user to choose to become either an organization user or a simple user. I think the `showOrgForm` has three values, `true`, `false` and (`undefined` or `null`). If the `showOrgForm` is explicitly set to false, the user is shown `"I want to create an organization"`. And if the `showOrgForm` is explicitly set to true, the user is shown `"I don't want to create an organization"`. If `showOrgForm` is not set to either true or false, the user is shown, `"I want to create an organization"`. Might need @thisuraseniya's opinion too.

Reply:
after reading the code I found there is only two possibilites either user want to see org form or he dont want to see the org form . So can we do like this we only set showOrgForm value to false and we will switch between true and false state, We will trigger setShowOrgForm when user click the button ?

Reply:
As I can remember just after the account is created, the value is `null`. BTW, can you double check it by creating a new profile?

Reply:
Better to check the passing value by using debug mode. Then we can see whether this null value is coming by default or from an issue.

Reply:
@Niweera  I double checked it with a second email id . Its value is null (console.log) when user succesfully login.

Reply:
So it is not set to true or false in the initial time.

Reply:
``` const [showOrgForm, setShowOrgForm] = useState(null); ``` yeah this line set the value null at initial .

Reply:
Then it seems like the logic is correct in my point of view.